check_buyable_vs_available.py

```
@@ -0,0 +1,97 @@
+}
+
+params = {
+    'CANO': cano,
+    'ACNT_PRDT_CD': acnt,
+    'OVRS_EXCG_CD': '',
+    'TR_CRCY_CD': 'USD',
+    'CTX_AREA_FK200': '',
+    'CTX_AREA_NK200': ''
+}
+
+r = requests.get(f'{base_url}/uapi/overseas-stock/v1/trading/inquire-balance',
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)